src/app/components/HoldPage/HoldConfirmation.jsx

```diff
@@ -43,14 +43,14 @@ class HoldConfirmation extends React.Component {
 
   render() {
     // Need to better clarify variable names later.
-    const item = this.props.bib;
-    const title = (item && _isArray(item.title) && item.title.length > 0) ?
-      item.title[0] : '';
-    const id = (item && item['@id'] && typeof item['@id'] === 'string') ?
-      item['@id'].substring(4) : '';
-    const itemId = (this.props.params && this.props.params.id) ? this.props.params.id : '';
-    const selectedItem = LibraryItem.getItem(item, itemId);
-    const location = LibraryItem.getLocation(item, itemId);
```

Comment:
Do we or will we still need the function `LibraryItem.getLocation()` if we are going to have multiple delivery locations now.